Fix managed authentication docs (#1250)
wso2/docs-choreo-dev: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md

     2. In the left navigation menu, click **Settings**.
     3. Click the **Authentication Keys** tab and then click on the environment for which you want to generate keys.
     4. In the **Identity Provider** list, select **Choreo Built-In Identity Provider**.
     5. Click **Generate Secret**. 

        !!! Note
             If the **Regenerate Secret** button is shown instead of the **Generate Secret** button, it indicates that OAuth keys are already generated for the component for the selected environment.

    !!! tip

>         If you need to invoke APIs secured with role-based access control, you must create roles in the application and map those roles to relevant permissions (scope). Then those roles should be assigned to user groups. For more information, see [Create roles and assign permissions](../test-secure-api-access-with-choreo-built-in-security-token-service/#step-2-create-roles-and-assign-permissions) and [Assign roles to user groups](../test-secure-api-access-with-choreo-built-in-security-token-service/#step-3-assign-roles-to-user-groups) sections in [Test Secure API Access with Choreo Built-In Security Token Service](../test-secure-api-access-with-choreo-built-in-security-token-service).

Reviewer: ```suggestion
        If you need to invoke APIs secured with role-based access control, you can test this within Choreo itself by creating roles for the application and mapping those roles to relevant permissions (scope) and user groups. For more information, see [Create roles and assign permissions](../test-secure-api-access-with-choreo-built-in-security-token-service/#step-2-create-roles-and-assign-permissions) and [Assign roles to user groups](../test-secure-api-access-with-choreo-built-in-security-token-service/#step-3-assign-roles-to-user-groups) sections in [Test Secure API Access with Choreo Built-In Security Token Service](../test-secure-api-access-with-choreo-built-in-security-token-service).
```

Author: Resolved with [b498cef](https://github.com/wso2/docs-choreo-dev/pull/1250/commits/b498cef69958e084f6abf79960aa5f3f33db550f)

         2. Add the following as authorized redirect URL.
         3. Specify the following as authorized redirect URLs:
         4. Specify the access token type as JWT.

>             !!! tip

Reviewer 2: this tip is duplicated isn't it?

Author: This is in a tab view. Similar tips with slight variations (Indicating how to perform the role to group mapping for each type of IdP) is displayed in each tab
